internal/infrastructure/market-strategy/pluggable/strategy.go

```diff
@@ -1,7 +1,8 @@
-package domain
```

Comment:
Maybe you're referring to pluggable strategy, not swap parser?

I actually moved this strategy here because it was in core/domain/ originally. I agree with you that since this doesn't depend on domain it can be moved in pkg. I'm moving it to pkg/marketmaking.

Reply:
for what concerns swap parser, instead, since it depends on domain, it's correct to have it in infrastructure, since it used proto stubs which are are a thing of our infra and also internal stuff like the Swap domain.

Reply:
done.

Reply:
I would define infrastructure as another process inside our deployment(DB etc.)

Reply:
you're not that wrong actually :).. so shall I move the swap-parser to application? we can't have it in pkg because it depends on domain which is an internal layer. Application seem the right package unless we create a another folder inside/outside core. What do you think? @sekulicd

Reply:
Yes

Reply:
Done.